classes/cargo.bbclass

```diff
@@ -89,6 +89,7 @@ oe_cargo_fix_env () {
 	export HOST_CC="${RUST_BUILD_CC}"
 	export HOST_CFLAGS="${RUST_BUILD_CFLAGS}"
 	export HOST_AR="${BUILD_AR}"
+	export RUST_SYSROOT="${STAGING_DIR_NATIVE}/${prefix_native}"
```

Comment:
Yes, we can always use this value as the sysroot.  Rust is smart enough to look in a ${RUST_SYSROOT}/${TARGET_SYS} when cross-compiling.

This could be moved to rust.bbclass, but it's not necessary.  That bbclass already throws the --sysroot command flag, which I find preferable due to requiring less action-at-a-distance.  Unfortunately that is required we cargo, as we have no way to get cargo to throw the --sysroot flag.